### ISSUE TITLE
FeaterMask doesn't seem to work

### ISSUE THREAD

**Issue by basvdelsen** [comfyanonymous/ComfyUI#1570]
My guess is it should blur the edges of a mask you send through it? But no matter what number is put in, there's no amount of feathering.

https://i.imgur.com/y0t1ikL.png

**Comment by mcmonkey4eva** [comment]
There *is* feathering it's just very stupid feathering - it literally feathers the square edges of the image space, not the actual mask itself.

**Comment by basvdelsen** [comment]
> There _is_ feathering it's just very stupid feathering - it literally feathers the square edges of the image space, not the actual mask itself.

Is that what featherMask does? That does seem to be bugged then. I also tried a gaussian blur node, which does blur the mask but it seems to throw an error related to the size, so Idk what that's about but I can't use it either.

**Comment by gergovari** [comment]
Gaussian blur works for me, indeed the current FeatherMask's name is quite misleading.

**Comment by Schwi-AI** [comment]
> Gaussian blur works for me, indeed the current FeatherMask's name is quite misleading.

Agree. Since blurring masks is such a common use case, I feel that ComfyUI should definitely provide a built-in implementation.
